### ISSUE TITLE
using --market or -m doesn't work

### ISSUE THREAD

**Issue by scrummie02** [shweshi/terminal-stocks#7]
installed from NPM and get the error:
```
 terminal-stocks -m

Sorry, we couldn't find. Please check the stock ticker and provide correct one.

 Cannot read property 'split' of undefined

```
Seems the -m flag doesn't work.

**Comment by chris-hmmr** [comment]
Hi @scrummie02, a pull request on this was provided here: https://github.com/shweshi/terminal-stocks/pull/11

**Comment by shweshi** [comment]
@scrummie02 Fixed by PR #12 Thanks to @chris-hmmr
